static/js/components/dashboard/courses/StatusMessages.js

```diff
@@ -88,6 +88,31 @@ const messageForNotAttemptedExam = (course: Course) => {
   return message
 }
 
+const messageForAttemptedExams = (course: Course, passedExam: boolean) => {
+  const whenFailed = passedExam ? "" : "You did not pass the exam. "
```

Comment:
why not handle passed exam message here

Reply:
what do you mean?

Reply:
just a question when user passed course we are showing "You passed this course.". 
Just saw in you PR description image. I was saying can we add/handle that message here?